CSharpBasics3/Homework_3.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Numerics;
+using Microsoft.VisualBasic;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Globalization;
+
+namespace Kristina_Kulich__Application.CSharpBasics3;
+
+public class Homework_3
+{
+  public void Task_1()
+  {
+    Console.WriteLine("Input number:");
+    int number = 0;
+    int.TryParse(Console.ReadLine(), out number);
+   
+    if (number % 2 == 0 && number % 5 == 0)
+    {
+      Console.WriteLine($"For number {number}: tutti-frutti");
+    }
+    else if (number % 2 == 0)
+    {
+      Console.WriteLine($"For number {number}: tutti");
+    }
+    else if (number % 5 == 0)
+    {
+      Console.WriteLine($"For number {number}: frutti");
+    }
+
+  }
+
+  public void Task_2()
+  {
+    Console.WriteLine("Input number: ");
+    int numberleast = Convert.ToInt32(Console.ReadLine());
+    Console.WriteLine("Input bigger number: ");
+    int numberbigger = Convert.ToInt32(Console.ReadLine());
+    
+
+    for (int i = numberleast; i < numberbigger; i++)
```

Comment:
It would be better to create private method for "for loop" and use it for Task_2 and Task_3

Reply:
fixed